.travis.yml

```diff
@@ -18,9 +18,13 @@ env:
     - TEST=clang-tidy-fix
     - TEST=clang-tidy-check
     - ROS_DISTRO=eloquent
+before_install:
+- git lfs install
+#- echo -e "machine github.com\n  login $GITHUB_TOKEN" > ~/.netrc
+- git lfs pull
 
 before_script:
-  - git clone -q -b ros2 --depth=1 https://github.com/ros-planning/moveit_ci.git .moveit_ci
+  - git clone -q -b ros2 --depth=1 https://github.com/nbbrooks/moveit_ci.git .moveit_ci
```

Comment:
Please revert this. I'm assuming you have a PR over there as well?

Reply:
I'm trying to debug several frustrating issues with moveit_ci right now which is why this is here